gulpfile.js

```diff
@@ -130,9 +129,6 @@ gulp.task('default', conf.tasks.default, () => {
 gulp.task('movecss', () =>
   gulp.src(conf.view.dir + conf.view.cssFiles)
     .pipe(gulp.dest(conf.view.dest))
-    .on('end', () => {
```

Comment:
what do we lose by not calling exit explicitly?

Reply:
Because gulp is already calling it. We are abruptly exiting. So we return to gulp then gulp will print log line that task xys is completed in xyz second and it will exit after that.

Reply:
![screen shot 2018-06-19 at 12 30 06 pm](https://user-images.githubusercontent.com/1593996/41619847-8bcf18cc-73bc-11e8-83e5-d0be27280be0.png)
see this.